scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
+
+// SQL injection pattern
+const query = `SELECT * FROM users WHERE id = ${process.argv[2]}` // Unsafe SQL
```

Comment:
:stop_sign: **Security Vulnerability**: SQL injection vulnerability pattern detected. This allows potential unauthorized database access[^4].
```suggestion
// Use parameterized queries to prevent SQL injection
const query = 'SELECT * FROM users WHERE id = ?'
const params = [process.argv[2]]
// Execute the query with params using your database library
// For example, with mysql:
// connection.query(query, params, (error, results, fields) => {
//     if (error) throw error
//     console.log(results)
// })
```